cmd/kosli/reportEvidenceCommitJira_test.go

```diff
@@ -5,8 +5,8 @@ import (
 	"os"
 	"testing"
 
-	"github.com/go-git/go-billy/v5"
-	"github.com/go-git/go-git/v5"
+	billy "github.com/go-git/go-billy/v5"
+	git "github.com/go-git/go-git/v5"
```

Comment:
@sami-alajrami why do we need to have these aliases?

Reply:
the code works without them. But my vs code got confused about v5 being there twice (a VS code restart solved it). But i put them back mainly for readability as they are used in lines 22, 23 and it can take sometime to trace them back to those dependencies if you are new to the code.